gravatar/src/main/java/com/gravatar/AvatarQueryOptions.kt

```diff
@@ -105,3 +106,23 @@ public class AvatarQueryOptions private constructor(
 public fun AvatarQueryOptions(initializer: AvatarQueryOptions.Builder.() -> Unit): AvatarQueryOptions {
     return AvatarQueryOptions.Builder().apply(initializer).build()
 }
+
+/**
+ * Function to convert the [AvatarQueryOptions] to a string with the query parameters.
+ */
+public fun AvatarQueryOptions?.asQueryParameters(): String {
```

Comment:
If we leave it as an extension I would keep it internal. My understanding is that this was added internally in the `AvatarUrl` so the the whole config happens by using `AvatarQueryOptions` and not manually adding the params to the string, right?